text/0000-javascript-module-api.md

```diff
@@ -0,0 +1,1151 @@
+| `Ember.NoneLocation`                  | `import NoneLocation from "@ember/routing/none-location"`                  |
+| `Ember.Object`                        | `import EmberObject from "@ember/object"`                                  |
+| `Ember.RSVP`                          | `import RSVP from "rsvp"`                                                  |
+| `Ember.Resolver`                      | `import Resolver from "@ember/application/resolver"`                       |
+| `Ember.Route`                         | `import Route from "@ember/routing/route"`                                 |
+| `Ember.Router`                        | `import Router from "@ember/routing/router"`                               |
+| `Ember.Service`                       | `import Service from "@ember/service"`                                     |
+| `Ember.String.camelize`               | `import { camelize } from "@ember/string"`                                 |
+| `Ember.String.capitalize`             | `import { capitalize } from "@ember/string"`                               |
+| `Ember.String.classify`               | `import { classify } from "@ember/string"`                                 |
+| `Ember.String.dasherize`              | `import { dasherize } from "@ember/string"`                                |
+| `Ember.String.decamelize`             | `import { decamelize } from "@ember/string"`                               |
+| `Ember.String.fmt`                    | `import { fmt } from "@ember/string"`                                      |
+| `Ember.String.htmlSafe`               | `import { htmlSafe } from "@ember/string"`                                 |
+| `Ember.String.loc`                    | `import { loc } from "@ember/string"`                                      |
+| `Ember.String.underscore`             | `import { underscore } from "@ember/string"`                               |
+| `Ember.String.w`                      | `import { w } from "@ember/string"`                                        |
+| `Ember.TextArea`                      | `import TextArea from "@ember/component/text-area"`                        |
+| `Ember.TextField`                     | `import TextField from "@ember/component/text-field"`                      |
+| `Ember.addListener`                   | `import { addListener } from "@ember/object/events"`                       |
+| `Ember.addObserver`                   | `import { addObserver } from "@ember/object/observers"`                    |
+| `Ember.aliasMethod`                   | `import { aliasMethod } from "@ember/object"`                              |
+| `Ember.assert`                        | `import { assert } from "@ember/debug"`                                    |
+| `Ember.assign`                        | `import { assign } from "@ember/polyfills"`                                |
+| `Ember.cacheFor`                      | `import { cacheFor } from "@ember/object/internals"`                       |
```

Comment:
http://emberjs.com/api/classes/Ember.html#method_compare seems likely for this list. Can we include it?